env/cloud/secrets/terragrunt.hcl

```diff
@@ -13,6 +13,8 @@ locals {
   notify_callback_bearer_token = get_env("NOTIFY_CALLBACK_BEARER_TOKEN", "I_am_not_a_secret_token")
   notify_api_key               = get_env("NOTIFY_API_KEY", "I_am_not_a_secret_token")
   freshdesk_api_key            = get_env("FRESHDESK_API_KEY", "I_am_not_a_secret_token")
+  zitadel_provider             = get_env("ZITADEL_PROVIDER", "I_am_not_a_secret_token")
+  zitadel_administration_key   = get_env("ZITADEL_ADMINISTRATION_KEY", "I_am_not_a_secret_token")
```

Comment:
Just checking that it's ok if the secrets get created in the Prod account with these values since we don't have an environment variable override in the Prod GitHub workflows.

Reply:
I don't see an issue. I purposely avoided defining the production variables until we deploy the production IDP.